From ab625957c7d169031b91bc7dba0d72bbee063fff Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Wed, 20 Jun 2018 10:22:32 +0530
Subject: [PATCH 17/22] staging: fsl_ppfe/eth: reorganize pfe_netdev_ops

Reorganize members of struct pfe_netdev_ops to match with the order
of members in struct net_device_ops defined in include/linux/netdevice.h

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>

[ Srinidhi Rao : Ported this patch to photon linux from
  qoriq-components linux 'linux-4.14-nxp'
  (https://source.codeaurora.org/external/qoriq/qoriq-components/linux)
]

Signed-off-by: srinidhira0 <srinidhir@vmware.com>
---
 drivers/staging/fsl_ppfe/pfe_eth.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/fsl_ppfe/pfe_eth.c b/drivers/staging/fsl_ppfe/pfe_eth.c
index 265045fcb008..0aa08d251b7c 100644
--- a/drivers/staging/fsl_ppfe/pfe_eth.c
+++ b/drivers/staging/fsl_ppfe/pfe_eth.c
@@ -2243,11 +2243,11 @@ static const struct net_device_ops pfe_netdev_ops = {
 	.ndo_stop = pfe_eth_close,
 	.ndo_start_xmit = pfe_eth_send_packet,
 	.ndo_select_queue = pfe_eth_select_queue,
-	.ndo_get_stats = pfe_eth_get_stats,
-	.ndo_set_mac_address = pfe_eth_set_mac_address,
 	.ndo_set_rx_mode = pfe_eth_set_multi,
-	.ndo_set_features = pfe_eth_set_features,
+	.ndo_set_mac_address = pfe_eth_set_mac_address,
 	.ndo_validate_addr = eth_validate_addr,
+	.ndo_get_stats = pfe_eth_get_stats,
+	.ndo_set_features = pfe_eth_set_features,
 };
 
 /* pfe_eth_init_one
-- 
2.14.2

